src/libstd/net/ip.rs

```diff
@@ -89,6 +89,9 @@ impl Ipv4Addr {
     }
 
     /// Returns true if this is a loopback address (127.0.0.0/8).
+    ///
+    /// This property is defined by RFC 6890
```

Comment:
Should we disambiguate this RFC from our normal RFCs? Do we want to provide a link?